lib/note-detail.jsx

```diff
@@ -107,13 +99,17 @@ export default React.createClass( {
 	},
 
 	renderEditable( divStyle ) {
-		const note = this.props.note;
+		const { content = '' } = this.props.note.data;
```

Comment:
if we're just destructuring one property _and_ still allowing for a `TypeError` from the nested access, let's just use `_.get` instead…

```js
import { get } from 'lodash';

const content = get( this.props, 'note.data.content', '' );
```